tasks/watch.js

```diff
@@ -64,7 +34,5 @@ function watchFiles() {
 
 exports.watch = series(
   build,
-  buildPL,
-  serve,
   watchFiles
 );
```

Comment:
I get the following when I run `gulp watch` in this branch. 
FYI - It appears to be erroring at the `lintJs` task. ([See notes above](https://github.com/uswds/uswds/pull/4603/files#r841797206))
```
[06:49:33] Starting 'watchFiles'...
[06:49:33] 'watchFiles' errored after 12 ms
[06:49:33] AssertionError [ERR_ASSERTION]: Task never defined: undefined
    at getFunction (/Users/aleadem/Dev/github/uswds/node_modules/undertaker/lib/helpers/normalizeArgs.js:21:9)
    at map (/Users/aleadem/Dev/github/uswds/node_modules/arr-map/index.js:20:14)
    at normalizeArgs (/Users/aleadem/Dev/github/uswds/node_modules/undertaker/lib/helpers/normalizeArgs.js:30:10)
    at Gulp.parallel (/Users/aleadem/Dev/github/uswds/node_modules/undertaker/lib/parallel.js:13:14)
    at watchFiles (/Users/aleadem/Dev/github/uswds/tasks/watch.js:22:7)
    at bound (node:domain:421:15)
    at runBound (node:domain:432:12)
    at asyncRunner (/Users/aleadem/Dev/github/uswds/node_modules/async-done/index.js:55:18)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
[06:49:33] 'watch' errored after 25 s
```

Reply:
Fixed in 4dc84275395c7575c27e5c3d258adfb2b29bee32.